api/graphql/schema.graphql

```diff
@@ -558,6 +559,7 @@ input PoolCandidateFilterInput {
     operationalRequirements: [OperationalRequirement] @builder(method: "App\\Models\\PoolCandidate@filterByOperationalRequirements")
     workRegions: [WorkRegion] @builder(method: "App\\Models\\PoolCandidate@filterByWorkRegions")
     pools: [PoolFilterInput] @builder(method: "App\\Models\\PoolCandidate@filterByPools")
+    status: PoolCandidateStatus @scope(name: "PoolCandidateStatus")
```

Comment:
This can be removed. Instead we'll add the `available` scope to the countPoolCandidates and searchPoolCandidates queries.